ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -180,6 +192,7 @@ _exit_modes (bounding_boxes * bdata)
   if (bdata->mode == TFLITE_SSD_BOUNDING_BOX) {
     /* properties_TFLite_SSD *data = &bdata->tflite_ssd; */
   } else if (bdata->mode == TF_SSD_BOUNDING_BOX) {
+  } else if (bdata->mode == OV_PERSON_DETECTION_BOUNDING_BOX) {
```

Comment:
Why do you need this line?

Reply:
@gichan-jang Please refer https://github.com/nnstreamer/nnstreamer/pull/2652#discussion_r472665486